server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/data/CassandraUsersRepositoryModule.java

```diff
@@ -39,6 +41,9 @@
 public class CassandraUsersRepositoryModule extends AbstractModule {
     @Override
     public void configure() {
+        bind(UsernameValidatorAggregator.class).in(Scopes.SINGLETON);
```

Comment:
Binding the same implementation in multiple modules -> extract this binding to a common module.